src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    if (propertyValues.includes(propertyValues[i], i + 1)) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    const objPropertyValue = robot[key];
+
+    if (!inverse.hasOwnProperty(objPropertyValue)) {
+      inverse[objPropertyValue] = key;
+    }
+  }
```

Comment:
```suggestion
  for (const key in robot) {
    const objPropertyValue = robot[key];

    if (inverse.hasOwnProperty(objPropertyValue)) {
      return null;
    }
    
    inverse[objPropertyValue] = key;
  }
```
like that there is no need to do another loop and another if statement, we can also omit `popertyValues`